server/src/http_server/webhooks/cookd.rs

```diff
@@ -0,0 +1,40 @@
+use axum::{extract::State, http::StatusCode, Json};
+use cja::{app_state::AppState as _, color_eyre::eyre::Context};
+use serde_json::{json, Value};
+
+use crate::{
+    http_server::{errors::WithStatus as _, ResponseResult},
+    AppState,
+};
+
+#[derive(serde::Deserialize, Clone)]
+struct Payload {
+    subdomain: String,
+    slug: String,
+    player_github_email: Option<String>,
+    player_github_username: Option<String>,
+    score: i32,
+}
+
+pub(crate) async fn handler(
+    State(state): State<AppState>,
+    Json(webhook_payload): Json<Value>,
+) -> ResponseResult<Value> {
+    let payload: Payload = serde_json::from_value(webhook_payload)
+        .context("Could not parse payload into expected JSON")
+        .with_status(StatusCode::UNPROCESSABLE_ENTITY)?;
+
+    let db_result = sqlx::query!(
+        r#"
+        INSERT INTO CookdWebhooks (subdomain, slug, player_github_email, player_github_username, score)
+        VALUES ($1, $2, $3, $4, $5) RETURNING cookd_webhook_id
+        "#,
+        payload.subdomain,
+        payload.slug,
+        payload.player_github_email,
+        payload.player_github_username,
+        payload.score,
```

Comment:
I'm currently using camel case for the webhook payload but can chance it to snake_case to save you some lines on transforming it. It seems like most people would prefer snake_case anyways?

Reply:
🤷  Not a big deal to me lol

You'll see in my next push that its just one line to tell it to serialize/deserilize to camelCase and I can keep my Rusty snake case variables 🎉